cmd/server/server.go

```diff
@@ -153,101 +154,114 @@ func (d *multiDeployer) BeginDeployments(ctx context.Context, app *app.BuiltAppl
 
 }
 */
+
 func runServe(ctx context.Context) error {
-	type envConfig struct {
-		ConfigFile         string `env:"CONFIG_FILE"`
-		WorkerPort         int    `env:"WORKER_PORT" default:"8081"`
-		DeployerAssumeRole string `env:"DEPLOYER_ASSUME_ROLE"`
-		CFTemplates        string `env:"CF_TEMPLATES"`
-		CallbackARN        string `env:"CALLBACK_ARN"`
-	}
-	cfg := envConfig{}
-	if err := envconf.Parse(&cfg); err != nil {
-		return err
-	}
+	g, ctx := errgroup.WithContext(ctx)
 
-	log.WithField(ctx, "PORT", cfg.WorkerPort).Info("Boot")
+	g.Go(runWorker(ctx))
```

Comment:
I'm taking the assumption for the moment that we want the binary running all of the handler methods and not just one set.